From 00fc1682dbfbd108c8920d7128d16321758efa61 Mon Sep 17 00:00:00 2001
From: dankcushions <chris.bransden@gmail.com>
Date: Tue, 11 Aug 2020 01:02:14 +0100
Subject: [PATCH] Force GLES 3.1

---
 retroarch.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/retroarch.c b/retroarch.c
index 6a88c31..8ebba98 100644
--- a/retroarch.c
+++ b/retroarch.c
@@ -18507,6 +18507,12 @@ static bool dynamic_request_hw_context(enum retro_hw_context_type type,
 
 #if defined(HAVE_OPENGLES3)
       case RETRO_HW_CONTEXT_OPENGLES_VERSION:
+         if (major == 3 && minor == 2)
+         {
+            RARCH_ERR("Requesting OpenGLES%u.%u context, but device only supports OpenGLES3.1.\n",
+                   major, minor);
+            return false;
+         }
          RARCH_LOG("Requesting OpenGLES%u.%u context.\n",
                major, minor);
          break;
-- 
2.8.4 (Apple Git-73)

